### PR TITLE
create, verify and show approval endpoints

### PR DESCRIPTION
**Pull request by imax-iva**
https://github.com/edenlabllc/ehealth.api/issues/3962

### CODE REVIEW

File: apps/api/test/api_web/controllers/approval_controller_test.exs [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+      if is_atom(k) do
+        {Atom.to_string(k), to_map(v)}
+      else
+        {k, to_map(v)}
+      end
+    end)
+    |> Enum.into(%{})
+  end
+
+  defp to_map(value) when is_list(value) do
```
Comment (BatrakM) [inline apps/api/test/api_web/controllers/approval_controller_test.exs]: same here

File: apps/api/test/api_web/controllers/approval_controller_test.exs [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    Enum.map(value, &to_map/1)
+  end
+
+  defp to_map(value) do
```
Comment (BatrakM) [inline apps/api/test/api_web/controllers/approval_controller_test.exs]: and here

File: apps/api/lib/api_web/views/approval_view.ex [hunk 3]
```diff
@@ -0,0 +1,9 @@
+defmodule Api.Web.ApprovalView do
+  @moduledoc false
+
+  use ApiWeb, :view
+
+  def render("empty.json", _) do
```
[the remark this answers is not in the document]
Comment (imax-iva) [reply apps/api/lib/api_web/views/approval_view.ex]: fixed

File: apps/api/test/api_web/controllers/approval_controller_test.exs (hunk 1 above)
Comment (imax-iva) [reply apps/api/test/api_web/controllers/approval_controller_test.exs]: fixed

File: apps/api/test/api_web/controllers/approval_controller_test.exs (hunk 2 above)
Comment (imax-iva) [reply apps/api/test/api_web/controllers/approval_controller_test.exs]: fixed

**Pull request merged**